### DISCUSSION TITLE
Missing CPU fan within fan control

### DISCUSSION THREAD

**Discussion by gandalfwfh** [Rem0o/FanControl.Releases#2339] [Q&A]
Hi,
I have 4 fans connected (each is  based of 3 fans cascaded) 1 on the fans is connected to the CPU 
fan header and it belongs to the AIO radiator.
the AIO pump is connected next to the CPU fan header. I can see all the fans and the AIO fan but not the CPU fan. in the bios I can see this fan and change the speed with no issues. this seems like something quite easy to solve but I am quite new with all this (just finished building the computer), any idea what I am missing here. 
<img width="954" alt="missing one" src="https://github.com/Rem0o/FanControl.Releases/assets/121752223/7d5dc5e6-71fe-4449-9cca-8a0d4b26eae8">
This is defiantly not fan control specific issue , I also don't see this fan using Libre hardware monitor.

**Comment by gandalfwfh** [comment]
installing Asus AI suite 3 all the fans are present and working except on fan control , both cpu fan and cpu opt fan are not showing in fan control 
![Screenshot 2024-03-08 053603](https://github.com/Rem0o/FanControl.Releases/assets/121752223/72c63ba9-4d9d-47b3-bd79-a3acce51b3d0)

**Comment by Rem0o** [comment]
> This is defiantly not fan control specific issue , I also don't see this fan using Libre hardware monitor.

You pretty much got the cause. What exactly do you want/expect out of this discussion?

**Reply by gandalfwfh** [reply]
unfortunately I didn't got the cause , its not working using fan control but works with another program which means its not an hardware issue. if I am trying to get your point , its ok than use other software.

**Reply by Rem0o** [reply]
I said that because you figured out it didn't work with LHM, and Fancontrol uses LHM internally for everything hardware related.

**Comment by gandalfwfh** [comment]
Disconnecting  CPU_OPT (gpu aio) fans solved the issue.
Thanks
